packages/core/src/app/address/StaticAddress.test.tsx

```diff
@@ -18,7 +18,7 @@ import StaticAddress, { StaticAddressProps } from './StaticAddress';
 
 jest.mock('@bigcommerce/checkout/ui', () => ({
     ...jest.requireActual('@bigcommerce/checkout/ui'),
-    IconPayPalConnectSmall: () => (<div data-test="pp-fastlane-icon">IconPayPalFastlaneSmall</div>)
+    IconPayPalConnectSmall: () => (<div data-test="pp-connect-icon">IconPayPalFastlaneSmall</div>)
```

Comment:
We don't need snapshot tests anymore, can we please refactor this test to get rid of enzyme.

Reply:
Let me make it in separate PR, since this feature is a bit urgent for us